build.gradle

```diff
@@ -12,7 +12,8 @@ repositories {
 dependencies {
   compile 'com.squareup:javapoet:1.1.0'
   compile 'com.google.auto.service:auto-service:1.0-rc2'
-  compile 'com.google.auto.value:auto-value:1.2-rc1'
+  compile 'com.google.auto.value:auto-value:1.2'
+  compile 'com.gabrielittner.auto.value:auto-value-extension-util:0.1.0'
```

Comment:
Are you guaranteeing API compatibility forward and backward with this? We all share the same classpath so binary incompatibilities will end up breaking builds.